src/test/resources/testng.xml

```diff
@@ -23,6 +23,17 @@
     </classes>
   </test>
 
+  <test name="1417_test">
+    <classes>
+      <class name="test.github1417.TestClass">
+        <parameter name="browsername" value="firefox"/>
```

Comment:
I prefer to have a test that constructs or use its own suite file because it is easier to run alone.